src/internal/components/pluggableVisualizations/areaChart/tests/PluggableAreaChart.spec.tsx

```diff
@@ -47,117 +47,6 @@ describe("PluggableAreaChart", () => {
 
-    it("should reuse one measure, only one category and one category as stack", async () => {
-        const areaChart = createComponent();
-
-        const expectedBuckets: IBucket[] = [
-            {
-                localIdentifier: "measures",
-                items: referencePointMocks.oneMetricAndManyCategoriesReferencePoint.buckets[0].items,
-            },
-            {
-                localIdentifier: "view",
-                items: referencePointMocks.oneMetricAndManyCategoriesReferencePoint.buckets[1].items.slice(
-                    0,
-                    1,
-                ),
-            },
-            {
-                localIdentifier: "stack",
-                items: referencePointMocks.oneMetricAndManyCategoriesReferencePoint.buckets[1].items.slice(
-                    1,
-                    2,
-                ),
-            },
-        ];
-        const expectedFilters: IFilters = {
-            localIdentifier: "filters",
-            items: referencePointMocks.oneMetricAndManyCategoriesReferencePoint.filters.items.slice(0, 2),
-        };
-
-        const extendedReferencePoint = await areaChart.getExtendedReferencePoint(
-            referencePointMocks.oneMetricAndManyCategoriesReferencePoint,
-        );
-        expect(extendedReferencePoint).toEqual({
-            buckets: expectedBuckets,
-            filters: expectedFilters,
-            uiConfig: uiConfigMocks.oneMetricAndManyCategoriesAreaUiConfig,
-            properties: {},
-        });
-    });
-
-    it("should reuse all measures, only one category and no stacks", async () => {
-        const areaChart = createComponent();
-
-        const expectedBuckets: IBucket[] = [
-            {
-                localIdentifier: "measures",
-                items: referencePointMocks.multipleMetricsAndCategoriesReferencePoint.buckets[0].items,
-            },
-            {
-                localIdentifier: "view",
-                items: referencePointMocks.multipleMetricsAndCategoriesReferencePoint.buckets[1].items.slice(
-                    0,
-                    1,
-                ),
-            },
-            {
-                localIdentifier: "stack",
-                items: [],
-            },
-        ];
-        const expectedFilters: IFilters = {
-            localIdentifier: "filters",
-            items: referencePointMocks.multipleMetricsAndCategoriesReferencePoint.filters.items.slice(0, 1),
-        };
-
-        const extendedReferencePoint = await areaChart.getExtendedReferencePoint(
-            referencePointMocks.multipleMetricsAndCategoriesReferencePoint,
-        );
-        expect(extendedReferencePoint).toEqual({
-            buckets: expectedBuckets,
-            filters: expectedFilters,
-            uiConfig: uiConfigMocks.multipleMetricsAndCategoriesAreaUiConfig,
-            properties: {},
-        });
-    });
-
-    it("should return reference point with Date in categories even it was as second item", async () => {
```

Comment:
Covered in https://github.com/gooddata/gooddata-react-components/pull/1030/files#diff-fe5a846a7ede8c4874d77c3b7706a3ddR332